zcash_client_backend/src/data_api.rs

```diff
@@ -1393,7 +1391,7 @@ pub trait WalletRead {
     /// This is equivalent to (but may be implemented more efficiently than):
     /// ```compile_fail
     /// Ok(
-    ///     if let Some(result) = self.get_transparent_receivers(account)?.get(address) {
```

Comment:
I think this can now be equivalent to just `Ok(self.get_transparent_receivers(account, true, true)?.get(address))`. (The purpose of the `else` branch was only to include ephemeral addresses, and setting `include_ephemeral` to true does that. We also include change addresses now, but that's correct. It could be clarified by saying "a given transparent receiver, which can also be an internal or ephemeral address, ...".)

Reply:
Ephemeral address support has been removed from `get_transparent_receivers` at @str4d's request, so this still needs the if/else.